feature-pack/pom.xml

```diff
@@ -3020,7 +3020,7 @@
 
         <dependency>
             <groupId>io.smallrye</groupId>
-            <artifactId>smallrye-metrics</artifactId>
+            <artifactId>smallrye-metrics-2.0</artifactId>
```

Comment:
I see you adding a license for this, but should the license for io.smallrye:smallrye-metrics-2.0 also be removed?

Reply:
sorry, that's not in the commit message but when I did this upgrade, I realised that the license for `io.smallrye:smallrye-metrics` was missing.

That's why the PR adds the license for `io.smallrye:smallrye-metrics-2.0` and does remove any other one.

Reply:
@psakar @dosoudil any idea?